### PR TITLE
Tag articles using sphinx tags

### PR DESCRIPTION
**Pull request by mnabywan**
Tag the articles using `sphinx-tags`. 

Build docs: https://epics--52.org.readthedocs.build/en/52/

### CODE REVIEW

File: build-system/configuring-vxworks-6_x.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Configuring vxWorks 6.x
 
+```{tags} user, advanced
```
Comment (ralphlange) [inline build-system/configuring-vxworks-6_x.md:3]: consider making that "developer, advanced"
as users don't handle the central system configuration

File: build-system/cross-compile-epics-and-a-ioc-to-an-old-x86-linux.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # How To cross compile EPICS and a IOC to an old x86 Linux system
 
+```{tags} user, advanced
```
Comment (ralphlange) [inline build-system/cross-compile-epics-and-a-ioc-to-an-old-x86-linux.md:3]: should be "developer, advanced"
as per our definition that someone building an IOC is a developer.

File: build-system/how-to-port-epics-to-a-new-os-architecture.rst [hunk 3]
```diff
@@ -1,6 +1,9 @@
 How To Port EPICS to a new OS/Architecture
 ==========================================
 
+```{tags} user
```
Comment (ralphlange) [inline build-system/how-to-port-epics-to-a-new-os-architecture.rst:4]: Porting to a new architecture is clearly "developer, advanced"

File: getting-started/channel-access-reach-multiple-soft-iocs-linux.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # How to Make Channel Access Reach Multiple Soft IOCs on a Linux Host
 
+
+```{tags} user
```
Comment (ralphlange) [inline getting-started/channel-access-reach-multiple-soft-iocs-linux.md:4]: change to "developer, advanced" - users won't have the privileges to do anything

File: process-database/how-to-find-which-ioc-provides-a-pv.rst [hunk 5]
```diff
@@ -1,6 +1,8 @@
 How to find which IOC provides a PV
 ===================================
 
+.. tags:: user
```
Comment (ralphlange) [inline process-database/how-to-find-which-ioc-provides-a-pv.rst:4]: make it broader: "beginner, user, developer"?

**Review by ralphlange — APPROVED** [review]
Please check and consider the suggestions.
Not a blocker, though. Can be adapted later on.

**Comment by mnabywan** [comment]
Thanks for comment, changes applied

**Pull request merged**